includes/Experiments.php

```diff
@@ -395,6 +395,17 @@ public function get_experiments() {
 				'description' => __( 'Enable the Coverr tab in the Third-party media tab.', 'web-stories' ),
 				'group'       => 'editor',
 			],
+			/**
+			 * Author: @sks
+			 * Issue: #32064081
+			 * Creation date: 2020-08-28
+			 */
+			[
+				'name'        => 'enableEyeDropper',
+				'label'       => __( 'enableEyeDropper', 'web-stories' ),
+				'description' => __( 'Enable eye dropper.', 'web-stories' ),
```

Comment:
It doesn't actually enable the eyedropper, it just shows the (still non-functional) button, so this text might be a bit misleading. Also, _"eyedropper"_ is spelled in [one word](https://en.wiktionary.org/wiki/eyedropper).